spring-cloud-dataflow-rest-client/pom.xml

```diff
@@ -22,5 +22,23 @@
 			<groupId>org.springframework.plugin</groupId>
 			<artifactId>spring-plugin-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>joda-time</groupId>
```

Comment:
Just an `FYI comment`: 
We should keep an eye on this with regards to Java8 adoption as a minimum requirement. In that case we should move over to that date-time implementation immediately.